css/style.css

```diff
@@ -16,4 +16,379 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Variables */
+:root {
+    --orange: #E0633A;
+    --faint-orange: rgba(224, 99, 58, 0.1);
+}
+
+
+/* Accessibility Styles */
+
+.hide-this {
+    visibility: hidden; 
+}
+
+.show-this {
+    visibility: visible;
+}
+
+fieldset {
+    all: unset;
+}
+
+legend.hide-this {
+    all: unset;
+    height: 0;
+}
+
+
+ /* Navigation Styles */
+
+.nav-flex-container {
+    display: flex;
+    justify-content: space-between;
+    height: 4em;
+    width: 80%;
+    margin: auto;
+}
+
+.logo {
+    height: 2.25em;
+    margin: auto 0;
+}
+
+.nav-ul {
+    list-style-type: none;
+    margin: auto 0;
+    padding: 0;
+    font-weight: 450;
+}
+
+.nav-li {
+    display: inline;
+    margin: 0 0.75em;
+
+}
+
+.nav-link {
+    text-decoration: none;
+    color: rgb(165, 165, 165);
+    /* font-size: .90em; */
```

Comment:
Avoid leaving commented-out code. If not needed, better to remove old/deprecated code.